misc/nginx/README.md

```diff
@@ -0,0 +1,70 @@
+# Running ansible-runner-service production container
+
+## WARNING: THIS IS A WORK IN PROGRESS!
+
+## Goal:
+Provide a container version of the ansible runner service running in 'prod' mode.
+This version uses Nginx and uwsgi to provide a production environment for the
+Ansible Runner Service Flask application.
+
+While lightweight and easy to use, Flask’s built-in server is not suitable for
+production as it doesn’t scale well, and also for performance and security issues.
+
+The container runs ansible runner service with bind mounts out to the local filesystem
+to provide runtime, playbooks and artifacts persistence. This approach allows you to
+install an rpm containing the playbooks specific to ceph or gluster, then map these
+through to the Container.
+
+The provided container uses CentOS with Python 3.6 (most of the development work is being done against Fedora, with Python3.6).
+
+
+## Local Preparation
+1. download the project to your home directory and untar/unzip
+2. create an archive of the project called ansible-runner-service.tar.gz and
+store it in the misc/docker directory
+3. set up you local environment to persist the config
+```
+ sudo mkdir /etc/ansible-runner-service
+ sudo mkdir -p /usr/share/ansible-runner-service/{artifacts,env,inventory, project}
```

Comment:
space between ", project}" cause me issues creating the directories

Reply:
Thank you for testing it Daniel, i will remove the extra space.!!

The aim of this first review was only to present the pieces and how i'm configuring them. In this way i can have a quick feedback and change things (even components) if something is not ok.

In next PR i will add the SSL support with the mutual TLS authentication that will solve the problem you see about the ssl version number.